src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -796,6 +796,69 @@ def test_fw_exception(fortran_writer):
     assert "Unsupported node 'Node' found" in str(excinfo.value)
 
 
+def test_fw_filecontainer_1(fortran_writer):
+    '''Check the FortranWriter class outputs nothing when a
+    FileContainer node with no content is found.
+
+    '''
+    file_container = FileContainer("None")
+    result = fortran_writer(file_container)
+    assert not result
+
+
+def test_fw_filecontainer_2(fortran_writer):
+    '''Check that an instance of the FortranWriter class outputs the
+    expected code when a FileContainer contains multiple nodes (in
+    this case a Container (module) and a Routine (subroutine).
+
+    '''
+    container = Container("mod_name")
+    routine = Routine("sub_name")
+    file_container = FileContainer.create("None", SymbolTable(), [container, routine])
```

Comment:
pycodestyle error line to long

Reply:
Oops, sorry. I thought I'd checked all the files.